pdns/pdns_recursor.cc

```diff
@@ -3744,7 +3722,12 @@ static void makeUDPServerSockets(deferredAdd_t& deferredAdds)
 
     setCloseOnExec(fd);
 
-    setSocketReceiveBuffer(fd, 250000);
+    try {
+      setSocketReceiveBuffer(fd, 25000);
```

Comment:
is this value supposed to change from before?

Reply:
Not at all, thanks a lot!